scripts/downloadTemplate.gradle

```diff
@@ -30,15 +37,15 @@ task downloadTemplate(
             template = templates.keySet()[(ant.template as Integer)-1]
         }
         println "${color 'green', 'Install '+template+' documentation template.'}"
-        def targetDir = new File(new File(docDir, inputPath), 'arc42')
+        def targetDir = new File(new File(docDir, inputPath), template)
         targetDir.mkdirs()
-        if (template=='arc42') {
-            println "For more information about arc42 see https://arc42.org"
-            /**
-             if (targetDir.exists()) {println "${targetDir.canonicalPath} already exists"
```

Comment:
you removed this, i think it was useful to let the user know that there is already a setup and the intentional re-install requires manually removal of the existing structure

Reply:
But if you think it is useful, we can re-enable it